cfep-22.md

```diff
@@ -0,0 +1,48 @@
+
+<table>
+<tr><td> Title </td><td> TB2C - Too Big To Be Compromised </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Matthew R. Becker &lt;becker.mr@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> April 12, 2024</td></tr>
+<tr><td> Updated </td><td> April 12, 2024</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+There are some feedstocks on `conda-forge` that are "too big to be compromised" (TB2C).  These feedstocks
+form core parts of the ecosystem and a malicious build by a malicious maintainer could have dire consequences
+for our users and the reputation of `conda-forge`.  This CFEP proposes a new policy for these feedstocks to provide
+additional controls on who can build and upload packages from them.
+
+## Specification
+
+The `conda-forge/core` team would maintain a list of TB2C feedstocks in a central, publicly accessible location.
+
+The following rules would be applied to these feedstocks and this list:
+
+- Only members of the `conda-forge/core` team, upstream maintainers of the software in the feedstock, or maintainers
+  approved by the `conda-forge/core` team (either through unanimous agreement of the existing `conda-forge/core` maintainers on
+  the feedstock or with a 50% sensitive vote of the full `conda-forge/core` team) would be allowed access to these feedstocks.
+- Any non-core maintainer can be removed from the feedstock by a 50% sensitive vote of the `conda-forge/core` team.
+- Code reviews are required for all PRs to these feedstocks.
+- No automerge by the autotick bot is allowed for these feedstocks. Automerge labels are allowed.
```

Comment:
I think this should be formulated in a more generic way. "Automated merges without human intervention are not allowed", in case the autotick bot is phased out or replaced with something else.